src/main/java/nextstep/subway/auth/ui/AuthenticatorStrategy.java

```diff
@@ -0,0 +1,11 @@
+package nextstep.subway.auth.ui;
+
+import nextstep.subway.auth.domain.Authentication;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public interface AuthenticatorStrategy {
```

Comment:
```suggestion
public interface Authenticator {
```

더 명확한 네이밍이 될 수 있을 것 같아요 :)